.gitlab-ci.yml

```diff
@@ -153,7 +153,7 @@ common-stg:
     - rake configure_serviceaccount_ci_restore
     - rake infra_init
     - rake apply_infra
-    - cd ../../../gcp/live/stg
+    - cd ../../../gcp/live/dev
```

Comment:
I don't understand this change.

I also think it's pretty unexpected to deploy an entire gcp/dev environment in the middle of a common/stg deployment. Why are we doing this? Is there a different approach we can take?

Reply:
The goal of the `stg` environment of the common part is the testing of the initialization of the organization, and whether all the resources created are able to support the dev/stg/prd environments. I think that a dev environment is the worst case scenario where multiple subdomains (also set by the initial resources) are involved. That's why a dev environment of a virtual user is created. We can also add the creation of a stg or prd environment if needed.
And all of the above avoiding any interaction with the real production organization (*.gcp.gpii.net).